group_vars/all.yml

```diff
@@ -280,3 +280,8 @@ manila_enabled: false
 
 #image path for overcloud nodes
 remote_image_path: https://sf.redhat.com
+
+#DVR is the default for ML2/OVN deployments for osp releases >= 16
+#change below variable to false to disable DVR
+#Note: if Neutron_backend == 'ovs' or osp_release < 16, then DVR won't be enabled irrespective of setting below variable to true/false
```

Comment:
Do not have this feature. DVR is supported for ML2/OVS also, and is also supported in OSP 13. So change line 335 in overcloud.yml to only check for dvr_enabled == false.
Also change line 344 in overcloud.yml similarly.

Reply:
> Do not have this feature. DVR is supported for ML2/OVS also, and is also supported in OSP 13. So change line 335 in overcloud.yml to only check for dvr_enabled == false. Also change line 344 in overcloud.yml similarly.

ack